src/App.jsx

```diff
@@ -1,40 +1,32 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tabs } from './components/Tabs';
 
 export const tabs = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">
-      Selected tab is Tab 1
-    </h1>
+export const App = () => {
+  const [goodTab, setGoodTab] = useState('Tab 1');
```

Comment:
<img width="618" alt="Screenshot 2023-08-25 at 17 51 53" src="https://github.com/mate-academy/react_tabs-js/assets/135232001/93bef470-66d7-4726-8bed-b2016f2eb7de">
Consider using a tab[0].id here and the name of the state should be changed to meet task requirements